projects/lib/src/pgngame.cpp

```diff
@@ -43,7 +43,7 @@ PgnStream& operator>>(PgnStream& in, PgnGame& game)
 	return in;
 }
 
-QTextStream& operator<<(QTextStream& out, const PgnGame& game)
+QTextStream& operator<<(QTextStream& out, PgnGame& game)
```

Comment:
I wouldn't remove the const. It's confusing when operator << modifies its input parameters.